star-ros2/README.md

```
@@ -40,15 +40,15 @@ map (RTAB-Map - global, corrected)
-- **Docker** (for containerized ROS2 environment)
-- **VS Code** with **Dev Containers** extension
-- **Hardware** (for deployment): Raspberry Pi 5, SPI/USB/video device passthrough
+- **Docker** (for containerized ROS2 environment) -- or --
+- **Raspberry Pi 5** with ROS2 Jazzy natively installed (see [docs/PI_DEPLOYMENT.md](../docs/PI_DEPLOYMENT.md))
+- **VS Code** with **Dev Containers** extension (devcontainer path only)
 
-### Setup
+### Setup -- Devcontainer (x86/macOS development)
 
 1. **Open the repository in VS Code**
    ```bash
```

markdownlint-cli2 (0.21.0)

[warning] Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)